Presentation.Web/Models/External/V2/Request/SystemUsage/BaseItSystemUsageWriteRequestDTO.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections.Generic;
+using Presentation.Web.Models.External.V2.Request.Generic;
+using Presentation.Web.Models.External.V2.Types.Shared;
+
+namespace Presentation.Web.Models.External.V2.Request.SystemUsage
+{
+    public class BaseItSystemUsageWriteRequestDTO
```

Comment:
Hmm. Ved ikke om "Base" er det rigtige ord her. Det er jo alle de ting man kan rette der ligger her? Men det er jo selvfølgelig det vi har gjort i de andre også for at adskille "Create" og "Update" DTO'er på en let måde.

Reply:
mja altså det er jo næsten rigtigt. Create DTO har 2 ekstra properties. Det er lidt som de andre du nævner og det er nok det bedste bud lige nu, men jeg kan se at jeg mangler "abstract"